src/app/components/shell/lower-sticky-note/lower-sticky-note.js

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
 import {pageWrapper} from '~/controllers/jsx-wrapper';
 import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
-import {usePutAway, useStickyData, useSeenCounter} from '../shared.jsx';
+import {usePutAway, useStickyData} from '../shared.jsx';
 import analytics from '~/helpers/analytics';
 import './lower-sticky-note.css';
 
-const SEEN_ENOUGH = 7;
```

Comment:
if this keeps getting added/taken away, maybe we could add something to the CMS?